### PR TITLE
fix(cssgrid): add ie polyfill

### PR DESCRIPTION
**Pull request by shmidt-i**

### CODE REVIEW

**Comment by iamstarkov** [comment]
late PRs, huh?

File: src/Atoms/CssGrid/CssGrid.tsx [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    return styles.join('\n');
+  }
+
+  if (isGutterPresented) {
+    // eslint-disable-next-line no-nested-ternary
+    const rowGutter = isNumber(gutter)
+      ? `${theme.spacing.unit(gutter)}px`
+      : isNumber(gutter.row)
+      ? `${theme.spacing.unit(gutter.row)}px`
+      : gutter.row;
```
Comment (iamstarkov) [inline src/Atoms/CssGrid/CssGrid.tsx]: formatting is off

File: src/Atoms/CssGrid/CssGrid.tsx [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+      // Simplest case just return initial display: -ms-grid
+      return styles.join('\n');
+    }
+    if (isColumnsPresented) {
+      styles.push(`-ms-grid-columns: ${templateColumnStyles};`);
+    }
+    if (isRowsPresented) {
+      styles.push(`-ms-grid-rows: ${templateRowsStyles};`);
+    }
+    return styles.join('\n');
```
Comment (JacobBlomgren) [inline src/Atoms/CssGrid/CssGrid.tsx]: The tests you added are very good, but I would like some tests for these as well

**Comment by shmidt-i** [comment]
How we can hide stuff on different screen sizes? Reordering is covered, check about hiding stuff

**Pull request merged**